.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Disable core.autocrlf's line ending conversion behavior to prevent test failures on Windows
+* text=lf
```

Comment:
Are you testing this on windows? I thought on windows you want git to checkout files with crlf so that it matches windows editors. Does the build produce crlf endings?

If going with lf, we probably want an `.editorconfig` file with that setting as well.

Reply:
Yes, my current environment is Windows 10 + Linux Subsystem

In general, yes, on Windows you should checkout files with crlf, however the build will use whatever line endings files are checked out with, resulting in failing tests.

It might be a better idea to normalize line endings when doing output instead of this, thoughts?

Reply:
In practice, the CRLF situation leads to many issues across platforms. All Windows editors support LF properly so I found it easier to enforce LF.

I use the following `.gitattributes` file:
```
# Enforce `lf` for text files (even on Windows)
text eol=lf
```

End the following `.editorconfig` setting:
```
end_of_line = lf
```

In `tsconfig.json`, you should also ensure that `"newLine": "lf"` is set.